tools/tuhi-kete.py

```diff
@@ -318,10 +303,7 @@ def _on_signal_received(self, proxy, sender, signal, parameters):
 
     def _on_name_vanished(self, connection, name):
         logger.error('Tuhi daemon went away')
-        try:
-            self.mainloop.quit()
-        except AttributeError:
-            pass
```

Comment:
the call to mainloop.quit() is then re-added in a later commit. In this later commit, the try/except is missing, so I do not think we should remove it in the first place